include/fmt/core.h

```diff
@@ -392,11 +399,8 @@ template <typename Char> class basic_string_view {
     the size with ``std::char_traits<Char>::length``.
     \endrst
    */
-#if __cplusplus >= 201703L  // C++17's char_traits::length() is constexpr.
-  FMT_CONSTEXPR
```

Comment:
And another observation, it's not about this PR, just these lines in general: probably `constexpr` can be used here, not `FMT_CONSTEXPR`, since it's already under the `__cplusplus >= 201703L` macro check.

Reply:
Looks like this actually can be applied to this PR too, since it's also used in newly introduced macro definition.